tests/snuba/api/serializers/test_group.py

```diff
@@ -304,14 +304,15 @@ def test_seen_stats(self):
         group_id, = set(e.group.id for e in events)
 
         group = Group.objects.get(id=group_id)
-        group.times_seen = 5
-        group.first_seen = self.week_ago
+        group.times_seen = 3
+        group.first_seen = self.week_ago - timedelta(days=5)
+        group.last_seen = self.week_ago
         group.save()
 
         # should use group columns when no environments arg passed
         result = serialize(group, serializer=GroupSerializerSnuba(environment_ids=[]))
-        assert result["count"] == "5"
-        assert result["lastSeen"] == group.last_seen
+        assert result["count"] == "3"
+        assert iso_format(result["lastSeen"]) == iso_format(self.min_ago)
```

Comment:
What doesn't first_seen need to be wrapped in iso_format as well? 

*Disclaimer* Did no investigatory work on how the fields are stored.

Reply:
It's just laziness that I copied elsewhere from the test tbh. `self.min_ago` is tz aware, but `lastSeen` isn't, so this works around that.